web/html/citation-elements/intro-bdi.md

```diff
@@ -23,8 +23,8 @@ aspects:
   - obscura
 
 links:
-  - '[MDN docs for bdi](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi){website}'
-  - '[MDN docs for User Agent](https://developer.mozilla.org/en-US/docs/Glossary/user_agent){website}'
+  - '[MDN docs for bdi](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/bdi){documentation}'
+  - '[MDN docs for User Agent](https://developer.mozilla.org/en-US/docs/Glossary/user_agent){documentation}'
```

Comment:
no need to say "MDN docs"
first title could be HTML <bdi> Element
second could be just User Agent